themes/src/main/resources/theme/keycloak.v2/account/src/app/content/linked-accounts-page/LinkedAccountsPage.tsx

```diff
@@ -185,9 +179,22 @@ class LinkedAccountsPage extends React.Component<LinkedAccountsPageProps, Linked
                         <DataListItemRow key={account.providerName}>
                             <DataListItemCells
                                 dataListCells={[
-                                    <DataListCell key='idp'><Stack><StackItem isFilled>{this.findIcon(account)}</StackItem><StackItem id={`${account.providerAlias}-idp-name`} isFilled><h2><strong>{account.displayName}</strong></h2></StackItem></Stack></DataListCell>,
-                                    <DataListCell key='label'><Stack><StackItem isFilled/><StackItem id={`${account.providerAlias}-idp-label`} isFilled>{this.label(account)}</StackItem></Stack></DataListCell>,
-                                    <DataListCell key='username' width={5}><Stack><StackItem isFilled/><StackItem id={`${account.providerAlias}-idp-username`} isFilled>{account.linkedUsername}</StackItem></Stack></DataListCell>,
+                                    <DataListCell key='idp'>
+                                        <Split>
+                                            <SplitItem className="pf-u-mr-sm">{this.findIcon(account)}</SplitItem>
+                                            <SplitItem className="pf-u-my-xs" isFilled><span id={`${account.providerAlias}-idp-name`}>{account.displayName}</span></SplitItem>
+                                        </Split>
+                                    </DataListCell>,
+                                    <DataListCell key='label'>
+                                        <Split>
+                                            <SplitItem className="pf-u-my-xs" isFilled><span id={`${account.providerAlias}-idp-label`}>{this.label(account)}</span></SplitItem>
+                                        </Split>
+                                    </DataListCell>,
+                                    <DataListCell key='username' width={5}>
+                                        <Split>
+                                            <SplitItem className="pf-u-my-xs" isFilled><span id={`${account.providerAlias}-idp-username`}>{account.linkedUsername}</span></SplitItem>
+                                        </Split>
+                                    </DataListCell>,
```

Comment:
You probably don't need these vertical margin utilities - are you reducing the margin? If so you could try the compact variant of the data list. (I'm not familiar with what you are implementing here though)

Reply:
this was mainly used to position a text element to center it vertically next to a logo image. I tried looking at the isCompact prop, but that looks to only get applied on the DataList component. Having that applied compacts all of the white space within each row - which we need in this scenario